pkg/database/audit_entry.go

```diff
@@ -78,16 +78,25 @@ func (db *Database) PurgeAuditEntries(maxAge time.Duration) (int64, error) {
 	return result.RowsAffected, result.Error
 }
 
-// ListAudits returns the list audit events which match the given criteria.
+// ListAllAudits returns the list audit events which match the given criteria.
 // Warning: This list may be large. Use Realm.Audits() to get users scoped to a realm.
-func (db *Database) ListAudits(p *pagination.PageParams, scopes ...Scope) ([]*AuditEntry, *pagination.Paginator, error) {
+func (db *Database) ListAllAudits(p *pagination.PageParams, scopes ...Scope) ([]*AuditEntry, *pagination.Paginator, error) {
+	return db.ListAudits(nil, p, scopes...)
+}
+
+// ListAudits returns the list audit events which match the given criteria.
+func (db *Database) ListAudits(r *Realm, p *pagination.PageParams, scopes ...Scope) ([]*AuditEntry, *pagination.Paginator, error) {
 	var entries []*AuditEntry
 
 	query := db.db.
 		Model(&AuditEntry{}).
 		Scopes(scopes...).
 		Order("created_at DESC")
 
+	if r != nil {
```

Comment:
Ah, I didn't explain myself much 😄 

In `realm.go`, I think we want to do this:

```go
func (r *Realm) ListAudits(db *Database, p *pagination.PageParams, scopes ...Scope) (whatever) {
  scopes = scopes.Append(func(db *gorm.DB) *gorm.DB {
    return db.Where("audit_entries.realm_id = ?", r.ID)
  })

  return db.ListAudits(p, scopes...)
}
```

Does that make sense?